src/coreclr/gcinfo/gcinfodumper.cpp

```diff
@@ -703,8 +716,57 @@ GcInfoDumper::EnumerateStateChangesResults GcInfoDumper::EnumerateStateChanges (
 #pragma message("Unimplemented for LOONGARCH64 yet.")
     assert(!"unimplemented on LOONGARCH yet");
 #elif defined(TARGET_RISCV64)
-#pragma message("Unimplemented for RISCV64 yet.")
-    assert(!"unimplemented on RISCV64 yet");
+    FILL_REGS(pCurrentContext->R0, 33);
+    FILL_REGS(pCallerContext->R0, 33);
```

Comment:
> I want to check the number is updated when you remove PC.

We stay with PC here since all other architectures have this logic.
Note, I removed PC only from `switch` in another source, not really connected to this code logic.

> However, I cannot see your previous commits.

The only changes I did is:
https://github.com/dotnet/runtime/compare/ebfbce12b56e40f36c197729700479e2f276eebb..ea435f756d5bf3dc01ded976da44cb00c5ac91e7